lib/utils/index.js

```diff
@@ -645,21 +645,9 @@ class Utils {
   }
   
   static stringifyObject(objects) {
-    if (Array.isArray(objects)) {
-      return objects.map(obj => {
-        if (!Array.isArray(obj) && Utils.isObject(obj)) {
-          return inspect(obj);
-        }
-
-        return obj;
-      });
-    }
-
-    if (Utils.isObject(objects)) {
-      return inspect(objects);
-    }
+    const objectString = Utils.isObject(objects) ? inspect(objects) : objects;
```

Comment:
```suggestion
    if (Utils.isObject(objects)) {
      return inspect(objects);
    }

    return objects;
```

always easier to read and debug, in my opinion ;)

Reply:
In that case we have to return something like this:

```js
    if (Utils.isObject(objects)) {
      return `"${inspect(objects)}"`;
    }
    return `"${objects}"`;

```

Without the quotes the current testcases fail. And I want to keep this logic in util rather than assertion.js. 
We do not need the quotes in case of objects I guess. What do you think?

Reply:
Ah yes, we can keep it here. In that case your version using the ternary operator works better.

Reply:
Cool